displays/SOC_velocity_graph.py

```diff
@@ -0,0 +1,67 @@
+import matplotlib.pyplot as plt
+import os.path
+import sys
+sys.path.append('../')
+
+from optimization.car_model import Car
+from soc.SoCEstimation import CoulombCounter
+
+def calculateSOCValues(v_profile, e_profile, distance, initial_soc):
+    car_model = Car()
+    soc_est = CoulombCounter()
+
+    soc_intervals = [initial_soc]
+    soc_est.set_SoC(initial_soc)
+
+    for i in range(len(v_profile) - 1):
+        energy_used = car_model.energy_used(v_profile[i: i + 2], e_profile[i: i + 2], distance[i: i +1])
+        #in SoC calculation, wanted units is W = Js^-1, cancelled out in function's conversion so using 1 
+        soc_est.discharge(energy_used / 1, 1, True)
```

Comment:
discharge() takes value in W and converts it to kWh but energy_used result is given in J. I believe the direct conversion for J to kWh is /3600000. for the conversion in discharge(), it multiplies by time before dividing by 3600000, so as long as the denom and time passed match it shouldn't matter. 

TL;DR i divided by 1 for completion sake since it is asking for W but it doesn't do anything... I can delete it if you want